src/modules/hunting/proxy.py

```diff
@@ -83,8 +76,8 @@ def execute(self):
         if "buildDate" in version_metadata:
             self.event.evidence = "build date: {}".format(version_metadata["buildDate"])
```

Comment:
Should this be publishing an event too?

Reply:
Not really, but perhaps we can join the two, no need to do the same api call twice